contracts/RealmDiamond/facets/NFTDisplayFacet.sol

```diff
@@ -0,0 +1,30 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.9;
+
+import "../../libraries/AppStorage.sol";
+
+contract NFTDisplayFacet is Modifiers {
+  event NFTDisplayStatusUpdated(address _token, uint256 _chainId, bool _allowed);
+  //  event NFTBlacklisted(address _token, uint256 _chainId);
+  error LengthMisMatch();
+
+  function whitelistNFTs(address[] calldata _tokens, uint256[] calldata _chainIds) external onlyOwner {
```

Comment:
We don't need to specify specific tokenIDS here -- the whitelisting is done on a contract-by-contract basis, not specific tokenIDs @Timidan